staging/src/k8s.io/client-go/plugin/pkg/client/auth/plugins_providers.go

```diff
@@ -1,26 +0,0 @@
-//go:build !providerless
-// +build !providerless
-
-/*
-Copyright 2016 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package auth
-
-import (
-	// Initialize client auth plugins for cloud providers.
-	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
These are just stubs that provide helpful error messages.  Why would we remove them?

Reply:
yeah, agreed. these don't add any dependencies and are helpful pointers for people with old kubeconfig files

Reply:
client-go already dropped cloud provider deps a while back: https://github.com/kubernetes/kubernetes/pull/110013/files#r871431867

Reply:
Ack, adding it back